libs/ledger/include/ledger/identifier.hpp

```diff
@@ -58,13 +57,11 @@ class Identifier
   ConstByteArray        name_space() const;
   ConstByteArray const &full_name() const;
   ConstByteArray        qualifier() const;
-  std::size_t           size() const;
   bool                  empty() const;
 
   Identifier GetParent() const;
 
   // Parsing
-  bool Parse(ConstByteArray const &name);
   bool Parse(ConstByteArray &&name);
```

Comment:
Recommend simply making this a value to support move or copy semantics

Reply:
Done